flipperdevices/Flipper-Android-App
# add subfolders count to file manager

**Background**

Current file manager doesn't display subitems count inside folders

**Changes**

- Add subitems count
- Add `ExtendedListingItem` for better scalability 
- Add `visible` parameter for PlaceholderConnecting so there will be animation when it's loaded
 
**Test plan**

- Open sample app
- Open folders and see items are loading and each after each without blocking the listing itself
